src/coreclr/jit/fginline.cpp

```diff
@@ -478,23 +478,22 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
                 m_madeChanges = true;
             }
         }
-        else if (tree->OperGet() == GT_ASG)
+        else if (tree->OperIs(GT_STORE_LCL_VAR))
```

Comment:
Same as tree->OperIsLocalStore right?

Reply:
No, `OperIsLocalStore` also includes `STORE_LCL_FLD`, though we won't see it here (for now). This branch only dealt with `ASG(LCL_VAR, ...)`.